tests/imap_integration.rs

```diff
@@ -145,12 +196,11 @@ fn inbox() {
         .any(|m| m == &imap::types::UnsolicitedResponse::Recent(2)));
 
     // let's see that we can also fetch the e-mails
-    let fetch = c.fetch("1", "(ALL UID)").unwrap();
+    let fetch = c.fetch("1", "(ENVELOPE INTERNALDATE UID)").unwrap();
```

Comment:
Why all these changes away from `ALL`?

Reply:
Cyrus IMAP doesn't support ALL

Reply:
That's super weird — it's in the base IMAP RFC 3501 after all! For `FETCH` it is [defined as equivalent](https://datatracker.ietf.org/doc/html/rfc3501#page-55) to
```
FLAGS INTERNALDATE RFC822.SIZE ENVELOPE
```
so maybe we should use that?

Reply:
I saw that as well that it was int he 3501 RFC, but not the earlier 2060 spec. both of which claim to  4rev1.

I changed it to be what was actually be checked in the tests vs everything.